### PR TITLE
Change box-shadow for the menus

### PR DESCRIPTION
**Pull request by parasharrajat**
### Fixed Issues

$ https://github.com/Expensify/App/issues/6729

### Tests |  QA Steps

1. Go to the app
2. Go to chat
3. Click on any popover menu (e.g. add an attachment or right click on the message to open context menu etc)

### Tested On

- [x] Web
- [ ] Mobile Web
- [x] Desktop
- [ ] iOS
- [x] Android

#### Web | Desktop

https://user-images.githubusercontent.com/24370807/146250507-f8f42f57-7577-4c75-a188-d3b83e289a34.mp4

#### Android

https://user-images.githubusercontent.com/24370807/146263478-d4eaff4a-1922-4ca1-90d9-014cf815b938.mp4

#### IOS
Screenshot 2021-12-18 00:43:59

#### Mobile Web
Screenshot 2021-12-18 00:47:29

### CODE REVIEW

**Comment by parasharrajat** [comment]
cc: @shawnborton

**Comment by shawnborton** [comment]
Looking good! Can you share a mobile screenshot to confirm that the shadow isn't being applied there?

Also, what's going on with your chat input? It looks broken?

File: src/styles/getModalStyles.js [hunk 1]
```diff
@@ -154,7 +154,7 @@ export default (type, windowDimensions, popoverAnchorPosition = {}, containerSty
                 borderColor: themeColors.border,
                 justifyContent: 'center',
                 overflow: 'hidden',
-                boxShadow: '0px 0px 10px 0px rgba(0, 0, 0, 0.025)',
+                boxShadow: '0px 4px 12px 0px rgba(0, 0, 0, 0.06)',
```
Comment (shawnborton) [inline src/styles/getModalStyles.js:157]: I wonder if we should store this as some kind of global style/variable in case we want to use it elsewhere.

Comment (parasharrajat) [reply src/styles/getModalStyles.js:157]: Sounds good.

**Comment by parasharrajat** [comment]
> Also, what's going on with your chat input? It looks broken?

It's due to the Grammarly extension.

**Comment by parasharrajat** [comment]
@shawnborton @puneetlath  Updated.

**Comment by shawnborton** [comment]
Hard to tell from your video, but just confirming that the shadow does not show on mobile? Or does it?

**Comment by parasharrajat** [comment]
> Hard to tell from your video, but just confirming that the shadow does not show on mobile? Or does it?

It does not. Code is configured that way.

**Review by shawnborton — APPROVED** [review]

**Comment by shawnborton** [comment]
Sounds good, thanks for confirming!

**Review by puneetlath — COMMENTED** [review]
@parasharrajat it looks good to me. However, can you add screenshots from the other two platforms for completeness? Then I'll merge. Thanks!

**Comment by parasharrajat** [comment]
On the way.

**Comment by parasharrajat** [comment]
@puneetlath ScreenShots updated for all platforms.

**Review by puneetlath — APPROVED** [review]
Thank you!

**Pull request merged**